Expensify/App
# chg: Workspace Edit Page

### Details

1. Change the Form Input Field.
2. Added the error message.
3. Refactor to fix the issue.

### Fixed Issues

$ https://github.com/Expensify/App/issues/5194
$ https://github.com/Expensify/App/issues/5783
$ https://github.com/Expensify/App/issues/5777

### Tests | QA Steps
1. Navigate to settings
2. Click on a workspace
3. Click on the workspace avatar to edit
4. Delete the name and save.
5. You should error below the field.
6. Now add a new Name and save and save should happen smoothly.

**Please test all the three issues attached.**

### Tested On


### Screenshots


#### Web | Desktop

![image](https://user-images.githubusercontent.com/24370807/133487486-10eb93ee-5a68-4830-acda-2588cf902e09.png)

#### Mobile Web

![Screenshot 2021-09-15 23:45:17](https://user-images.githubusercontent.com/24370807/133487548-bf6041d7-dddf-4da3-a7ad-9bf995a906fd.png)

#### iOS


#### Android

![Screenshot 2021-09-15 23:45:17](https://user-images.githubusercontent.com/24370807/133487632-21b254f1-93de-4ff2-b101-b4af242c700f.png)
